engines/ospray/ispc/render/ExtendedOBJMaterial.ih

```diff
@@ -28,6 +28,14 @@
 #include <ospray/SDK/texture/TextureParam.ih>
 #include <ospray/SDK/volume/Volume.ih>
 
+// Caution: This has to match the Brayns::MaterialShadingMode
+enum MaterialShadingMode
```

Comment:
Can you not put this in a header file somewhere and include that in ispc and cpp code?

Reply:
I tried, but could not find a way to do so... I am having another look now, with not much hope though.

Reply:
I did something similar in brayns/common/geometry/Cone.h